package-testing/pre/000_install_package.rb

```diff
@@ -6,6 +6,10 @@
       # BKR-1109 requests a neater way to install an MSI
       msi_url = "http://#{ENV['BUILD_SERVER']}/pdk/#{ENV['SHA']}/repos/windows/pdk-x64.msi"
       generic_install_msi_on(workstation, msi_url)
+    elsif ENV['LOCAL_PKG']
+      pkg_fullname = File.basename(ENV['LOCAL_PKG'])
+      scp_to(workstation, ENV['LOCAL_PKG'], "/root/#{pkg_fullname}")
+      workstation.install_package(pkg_fullname)
```

Comment:
Either Windows should be supported; or the run should abort if LOCAL_PKG is attempted when workstation is Windows